Sources/CardPayments/CardClient.swift

```diff
@@ -225,17 +225,17 @@ public class CardClient: NSObject {
     }
 
     private func notifyCheckoutSuccess(for result: CardResult, completion: (CardResult?, CoreSDKError?) -> Void) {
-        analyticsService?.sendEvent("card-payments:3ds:succeeded")
+        analyticsService?.sendEvent("card-payments:approve-order:succeeded")
         completion(result, nil)
     }
 
     private func notifyCheckoutFailure(with error: CoreSDKError, completion: (CardResult?, CoreSDKError?) -> Void) {
-        analyticsService?.sendEvent("card-payments:3ds:failed")
+        analyticsService?.sendEvent("card-payments:approve-order:failed")
         completion(nil, error)
     }
 
     private func notifyCheckoutCancelWithError(with error: CoreSDKError, completion: (CardResult?, CoreSDKError?) -> Void) {
-        analyticsService?.sendEvent("card-payments:3ds:challenge:user-canceled")
+        analyticsService?.sendEvent("card-payments:approve-order:challenge:user-canceled")
```

Comment:
This one seems like it deviates a bit from the rest. Everything else seems to have the format `<MODULE>:<FEATURE>:<EVENT_NAME>`. Would `card-payments:approve-order:user-canceled-challenge` work?

Reply:
I was thinking of changing both "card-payments:vault-wo-purchase:challenge:canceled" and this one to 
"card-payments: approve-order:auth-challenge:canceled"
"card-payments: vault-wo-purchase:auth-challenge:canceled"

Reply:
Ok cool. This makes sense for now 👍

Reply:
Ty, Steven. Yeah, checking with analytics folks to coordinate name changing.
It totally makes sense to use approve-order. But I wanted to make all the final changes in a different PR.